src/modules/discovery/__init__.py

```diff
@@ -4,4 +4,5 @@
 # dynamically importing all modules in folder
 files = glob.glob(dirname(__file__)+"/*.py")
 for module_name in (basename(f)[:-3] for f in files if isfile(f) and not f.endswith('__init__.py')):
-    exec('from {} import *'.format(module_name))
\ No newline at end of file
+    if not module_name.startswith('test_'):
```

Comment:
I suspect this may be unnecessary now due to #96 as well

Reply:
Ha, this might have been a cleverer way of fixing #96!